### PR TITLE
Fix some syntax errors, remove unused include

### PR DESCRIPTION
**Pull request by betatim**
I removed the `k8s.txt` include because it was broken (and not being included at the moment). From the contents of `k8s.txt` and where it was being included I couldn't quite work out what should be included/in `k8s.txt`.

This also sets up travis so it runs the docs build as part of a PR to make sure we don't break the docs and don't notice.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -18,9 +18,12 @@ install:
 - pip install --no-cache-dir . -r dev-requirements.txt
 - npm install
 - npm run webpack
+# sphinx requirements
+- pip install --no-cache-dir . -r docs/doc-requirements.txt
```
Comment (minrk) [inline .travis.yml]: We should remove the `--no-cache` here. That's only for docker. There's also no `docs/doc-requirements.txt`, but there is a `docs/environment.yml`.

Comment (betatim) [reply .travis.yml]: but there is `doc/doc-requirements.txt`, fixed that typo.

Comment (betatim) [reply .travis.yml]: I find it annoying/confusing that we have both in the docs. It always leaves me guessing which should I use, who uses which file etc :_/

**Comment by minrk** [comment]
> sphinx-build: error: unrecognized arguments: -M

**Comment by betatim** [comment]
Huzzah!

**Pull request merged**